### DISCUSSION TITLE
Social Media Management

### DISCUSSION THREAD

**Discussion by loftwah** [EddieHubCommunity/LinkFree#8151] [Ideas]
This might be a little bit too large but I think this could have some pretty awesome potential too. The idea would be to make the LinkFree Dashboard a hub for getting and potentially managing their resources on the internet.

![linkfreesocial](https://github.com/EddieHubCommunity/LinkFree/assets/19922556/f78f1bb4-427a-4f73-b4f2-f33a27931c96)

**Comment by eddiejaoude** [comment]
Oh wow this might be a bit too big, but I have the idea of people selecting their best content for socials including GitHub projects, then this will preview and they could pin one one item to their profile
